[2025-01-27T23:29:48.923-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-27T23:29:48.943-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: InsertPACTRATxAGENDIntoElastic.inject_pactrat_agend_into_es scheduled__2021-05-29T00:00:00+00:00 [queued]>
[2025-01-27T23:29:48.962-0300] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: InsertPACTRATxAGENDIntoElastic.inject_pactrat_agend_into_es scheduled__2021-05-29T00:00:00+00:00 [queued]>
[2025-01-27T23:29:48.963-0300] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-27T23:29:49.005-0300] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): inject_pactrat_agend_into_es> on 2021-05-29 00:00:00+00:00
[2025-01-27T23:29:49.016-0300] {standard_task_runner.py:72} INFO - Started process 68743 to run task
[2025-01-27T23:29:49.019-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'InsertPACTRATxAGENDIntoElastic', 'inject_pactrat_agend_into_es', 'scheduled__2021-05-29T00:00:00+00:00', '--job-id', '7498', '--raw', '--subdir', 'DAGS_FOLDER/DAGInjectPACTRATXAGENDElastic.py', '--cfg-path', '/tmp/tmp4ke7f7c5']
[2025-01-27T23:29:49.021-0300] {standard_task_runner.py:105} INFO - Job 7498: Subtask inject_pactrat_agend_into_es
[2025-01-27T23:29:49.079-0300] {task_command.py:467} INFO - Running <TaskInstance: InsertPACTRATxAGENDIntoElastic.inject_pactrat_agend_into_es scheduled__2021-05-29T00:00:00+00:00 [running]> on host lucas-Aspire-A514-54
[2025-01-27T23:29:49.202-0300] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='lucas' AIRFLOW_CTX_DAG_ID='InsertPACTRATxAGENDIntoElastic' AIRFLOW_CTX_TASK_ID='inject_pactrat_agend_into_es' AIRFLOW_CTX_EXECUTION_DATE='2021-05-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-05-29T00:00:00+00:00'
[2025-01-27T23:29:49.203-0300] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-27T23:29:50.249-0300] {logging_mixin.py:190} WARNING - /home/lucas/airflow/dags/DAGInjectPACTRATXAGENDElastic.py:138 DtypeWarning: Columns (2) have mixed types. Specify dtype option on import or set low_memory=False.
[2025-01-27T23:29:50.855-0300] {_transport.py:349} INFO - DELETE http://localhost:9200/pacientes_agendamento [status:200 duration:0.121s]
[2025-01-27T23:29:51.489-0300] {_transport.py:349} INFO - PUT http://localhost:9200/pacientes_agendamento [status:200 duration:0.623s]
[2025-01-27T23:29:51.509-0300] {logging_mixin.py:190} INFO - {'acknowledged': True, 'shards_acknowledged': True, 'index': 'pacientes_agendamento'}
[2025-01-27T23:29:51.774-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.206s]
[2025-01-27T23:29:51.811-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.028s]
[2025-01-27T23:29:51.834-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.022s]
[2025-01-27T23:29:51.877-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.042s]
[2025-01-27T23:29:51.907-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.029s]
[2025-01-27T23:29:51.928-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.020s]
[2025-01-27T23:29:51.971-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.042s]
[2025-01-27T23:29:51.999-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.027s]
[2025-01-27T23:29:52.015-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.014s]
[2025-01-27T23:29:52.035-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.017s]
[2025-01-27T23:29:52.052-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.016s]
[2025-01-27T23:29:52.088-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.035s]
[2025-01-27T23:29:52.109-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.020s]
[2025-01-27T23:29:52.122-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.012s]
[2025-01-27T23:29:52.147-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.024s]
[2025-01-27T23:29:52.186-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.038s]
[2025-01-27T23:29:52.223-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.036s]
[2025-01-27T23:29:52.295-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.071s]
[2025-01-27T23:29:52.529-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.218s]
[2025-01-27T23:29:52.583-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.052s]
[2025-01-27T23:29:52.641-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.057s]
[2025-01-27T23:29:52.687-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.045s]
[2025-01-27T23:29:52.723-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.035s]
[2025-01-27T23:29:52.778-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.039s]
[2025-01-27T23:29:52.843-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.028s]
[2025-01-27T23:29:52.887-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.043s]
[2025-01-27T23:29:52.912-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.019s]
[2025-01-27T23:29:52.933-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.019s]
[2025-01-27T23:29:52.970-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.036s]
[2025-01-27T23:29:52.989-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.018s]
[2025-01-27T23:29:53.028-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.038s]
[2025-01-27T23:29:53.042-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.012s]
[2025-01-27T23:29:53.054-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.012s]
[2025-01-27T23:29:53.065-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.009s]
[2025-01-27T23:29:53.085-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.020s]
[2025-01-27T23:29:53.153-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.066s]
[2025-01-27T23:29:53.168-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.015s]
[2025-01-27T23:29:53.187-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.017s]
[2025-01-27T23:29:53.201-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.013s]
[2025-01-27T23:29:53.213-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.011s]
[2025-01-27T23:29:53.232-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.017s]
[2025-01-27T23:29:53.271-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.038s]
[2025-01-27T23:29:53.308-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.032s]
[2025-01-27T23:29:53.319-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.010s]
[2025-01-27T23:29:53.329-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.009s]
[2025-01-27T23:29:53.340-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.009s]
[2025-01-27T23:29:53.366-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.025s]
[2025-01-27T23:29:53.381-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.013s]
[2025-01-27T23:29:53.393-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.011s]
[2025-01-27T23:29:53.415-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.020s]
[2025-01-27T23:29:53.440-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.024s]
[2025-01-27T23:29:53.477-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.036s]
[2025-01-27T23:29:53.493-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.014s]
[2025-01-27T23:29:53.531-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.036s]
[2025-01-27T23:29:53.550-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.018s]
[2025-01-27T23:29:53.596-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.045s]
[2025-01-27T23:29:53.617-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.019s]
[2025-01-27T23:29:53.665-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.042s]
[2025-01-27T23:29:53.702-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.035s]
[2025-01-27T23:29:53.865-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.162s]
[2025-01-27T23:29:54.037-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.114s]
[2025-01-27T23:29:54.084-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.046s]
[2025-01-27T23:29:54.159-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.074s]
[2025-01-27T23:29:54.205-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.044s]
[2025-01-27T23:29:54.241-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.035s]
[2025-01-27T23:29:54.261-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.019s]
[2025-01-27T23:29:54.283-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.021s]
[2025-01-27T23:29:54.303-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.018s]
[2025-01-27T23:29:54.334-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.030s]
[2025-01-27T23:29:54.359-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.023s]
[2025-01-27T23:29:54.400-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.040s]
[2025-01-27T23:29:54.438-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.037s]
[2025-01-27T23:29:54.492-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.053s]
[2025-01-27T23:29:54.532-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.036s]
[2025-01-27T23:29:54.577-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.044s]
[2025-01-27T23:29:54.604-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.027s]
[2025-01-27T23:29:54.628-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.022s]
[2025-01-27T23:29:54.660-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.031s]
[2025-01-27T23:29:54.675-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.015s]
[2025-01-27T23:30:01.468-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:6.792s]
[2025-01-27T23:30:07.124-0300] {job.py:229} INFO - Heartbeat recovered after 12.89 seconds
[2025-01-27T23:30:17.520-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:9.892s]
[2025-01-27T23:30:18.120-0300] {job.py:229} INFO - Heartbeat recovered after 13.07 seconds
[2025-01-27T23:30:18.999-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:1.310s]
[2025-01-27T23:30:20.851-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:0.622s]
[2025-01-27T23:30:24.465-0300] {_transport.py:349} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:201 duration:2.169s]
[2025-01-27T23:30:38.036-0300] {job.py:229} INFO - Heartbeat recovered after 20.12 seconds
[2025-01-27T23:30:44.602-0300] {_transport.py:372} INFO - POST http://localhost:9200/pacientes_agendamento/_doc [status:N/A duration:10.144s]
[2025-01-27T23:30:44.859-0300] {_node_pool.py:249} WARNING - Node <Urllib3HttpNode(http://localhost:9200)> has failed for 1 times in a row, putting on 1 second timeout
[2025-01-27T23:30:46.395-0300] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/urllib3/connectionpool.py", line 534, in _make_request
    response = conn.getresponse()
               ^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/urllib3/connection.py", line 516, in getresponse
    httplib_response = super().getresponse()
                       ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/http/client.py", line 1428, in getresponse
    response.begin()
  File "/usr/lib/python3.12/http/client.py", line 331, in begin
    version, status, reason = self._read_status()
                              ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/http/client.py", line 292, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/socket.py", line 707, in readinto
    return self._sock.recv_into(b)
           ^^^^^^^^^^^^^^^^^^^^^^^
TimeoutError: timed out

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elastic_transport/_node/_http_urllib3.py", line 167, in perform_request
    response = self.pool.urlopen(
               ^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/urllib3/connectionpool.py", line 841, in urlopen
    retries = retries.increment(
              ^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/urllib3/util/retry.py", line 449, in increment
    raise reraise(type(error), error, _stacktrace)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/urllib3/util/util.py", line 39, in reraise
    raise value
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/urllib3/connectionpool.py", line 787, in urlopen
    response = self._make_request(
               ^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/urllib3/connectionpool.py", line 536, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=read_timeout)
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/urllib3/connectionpool.py", line 367, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPConnectionPool(host='localhost', port=9200): Read timed out. (read timeout=9.999809687999004)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow/dags/DAGInjectPACTRATXAGENDElastic.py", line 155, in inject_data_into_es
    generator(df, index, es_client)
  File "/home/lucas/airflow/dags/DAGInjectPACTRATXAGENDElastic.py", line 72, in generator
    client.index(
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elasticsearch/_sync/client/utils.py", line 455, in wrapped
    return api(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elasticsearch/_sync/client/__init__.py", line 2572, in index
    return self.perform_request(  # type: ignore[return-value]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elasticsearch/_sync/client/_base.py", line 271, in perform_request
    response = self._perform_request(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elasticsearch/_sync/client/_base.py", line 316, in _perform_request
    meta, resp_body = self.transport.perform_request(
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elastic_transport/_transport.py", line 342, in perform_request
    resp = node.perform_request(
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elastic_transport/_node/_http_urllib3.py", line 202, in perform_request
    raise err from e
elastic_transport.ConnectionTimeout: Connection timed out
[2025-01-27T23:30:57.829-0300] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=InsertPACTRATxAGENDIntoElastic, task_id=inject_pactrat_agend_into_es, run_id=scheduled__2021-05-29T00:00:00+00:00, execution_date=20210529T000000, start_date=20250128T022948, end_date=20250128T023057
[2025-01-27T23:30:58.040-0300] {job.py:229} INFO - Heartbeat recovered after 23.34 seconds
[2025-01-27T23:30:58.438-0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-27T23:30:58.508-0300] {standard_task_runner.py:124} ERROR - Failed to execute job 7498 for task inject_pactrat_agend_into_es (Connection timed out; 68743)
Traceback (most recent call last):
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/urllib3/connectionpool.py", line 534, in _make_request
    response = conn.getresponse()
               ^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/urllib3/connection.py", line 516, in getresponse
    httplib_response = super().getresponse()
                       ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/http/client.py", line 1428, in getresponse
    response.begin()
  File "/usr/lib/python3.12/http/client.py", line 331, in begin
    version, status, reason = self._read_status()
                              ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/http/client.py", line 292, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/socket.py", line 707, in readinto
    return self._sock.recv_into(b)
           ^^^^^^^^^^^^^^^^^^^^^^^
TimeoutError: timed out

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elastic_transport/_node/_http_urllib3.py", line 167, in perform_request
    response = self.pool.urlopen(
               ^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/urllib3/connectionpool.py", line 841, in urlopen
    retries = retries.increment(
              ^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/urllib3/util/retry.py", line 449, in increment
    raise reraise(type(error), error, _stacktrace)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/urllib3/util/util.py", line 39, in reraise
    raise value
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/urllib3/connectionpool.py", line 787, in urlopen
    response = self._make_request(
               ^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/urllib3/connectionpool.py", line 536, in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=read_timeout)
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/urllib3/connectionpool.py", line 367, in _raise_timeout
    raise ReadTimeoutError(
urllib3.exceptions.ReadTimeoutError: HTTPConnectionPool(host='localhost', port=9200): Read timed out. (read timeout=9.999809687999004)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow/dags/DAGInjectPACTRATXAGENDElastic.py", line 155, in inject_data_into_es
    generator(df, index, es_client)
  File "/home/lucas/airflow/dags/DAGInjectPACTRATXAGENDElastic.py", line 72, in generator
    client.index(
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elasticsearch/_sync/client/utils.py", line 455, in wrapped
    return api(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elasticsearch/_sync/client/__init__.py", line 2572, in index
    return self.perform_request(  # type: ignore[return-value]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elasticsearch/_sync/client/_base.py", line 271, in perform_request
    response = self._perform_request(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elasticsearch/_sync/client/_base.py", line 316, in _perform_request
    meta, resp_body = self.transport.perform_request(
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elastic_transport/_transport.py", line 342, in perform_request
    resp = node.perform_request(
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/lucas/airflow_env/lib/python3.12/site-packages/elastic_transport/_node/_http_urllib3.py", line 202, in perform_request
    raise err from e
elastic_transport.ConnectionTimeout: Connection timed out
[2025-01-27T23:30:58.666-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-01-27T23:30:58.863-0300] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-27T23:30:58.864-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
